.rubocop.yml

```diff
@@ -0,0 +1,138 @@
+AllCops:
+  TargetRubyVersion: 3.3
+  SuggestExtensions: false
+  # RuboCop has a bunch of cops enabled by default. This setting tells RuboCop
+  # to ignore them, so only the ones explicitly set in this file are enabled.
+  DisabledByDefault: true
+  Exclude:
+    - 'node_modules/**/*'
```

Comment:
@nomadicoder Should these be double quotes since that is the syntax we follow in all of our other ruby projects?